composer.json

```diff
@@ -13,14 +13,14 @@
     "require": {
         "php": ">=5.3.9",
         "symfony/framework-bundle": "~2.3",
-        "doctrine/phpcr-bundle": "1.2.*",
-        "doctrine/phpcr-odm": "1.2.*",
-        "symfony-cmf/core-bundle": "1.2.*",
-        "symfony-cmf/content-bundle": "1.2.*",
+        "doctrine/phpcr-bundle": "1.3.*",
+        "doctrine/phpcr-odm": "1.3.*",
+        "symfony-cmf/core-bundle": "1.3.*",
+        "symfony-cmf/content-bundle": "1.3.*",
         "symfony-cmf/menu-bundle": "1.2.*",
         "symfony-cmf/block-bundle": "1.2.*",
```

Comment:
so this is really causing trouble:

* `symfony-cmf/block-bundle` 1.3 requires `sonata-project/block-bundle: 2.3.*`
* `sonata-project/admin-bundle` 2.3 requires `sonata-project/block-bundle: 2.2.*`
* `sonata-project/admin-bundle` 2.4-dev is not supported by the CMF

In other words: We can't use CmfBlockBundle 1.3 when using the SonataAdminBundle with the next CMF version.